### ISSUE TITLE
Bed heat up crash

### ISSUE THREAD

**Issue by Mr-Pewpew99** [EverlastEngineering/Malyan-M200#2]
When bed and nozzle need to heat up, printer crashes and restarts. Nozzle only or bed only heat up works.

-Printer: Monoprice Select mini v2
-Firmware: v58.158.2

**Comment by EverlastEngineering** [comment]
I suspect the power supply cannot sustain the current draw necessary for both heaters, so the voltage drops causing the MCU to reboot. I recall reading that some units shipped with lower wattage supplies for north american import reasons. 

Any chance you could get a voltmeter on the 12V supply while doing this to confirm?

**Comment by Mr-Pewpew99** [comment]
You seem to be right, voltage drops from 12.35v no load to 11.52v with only nozzle or bed heating.
When I try to heat both it drops to 2v.

I bought my printer second hand with a 12v 5A supply, what power supply rating do you recommend for it to function properly with your firmware?

**Comment by EverlastEngineering** [comment]
Well at least this makes sense. I recommend an 8A supply, it should peak around 6 or 7, but even that can vary since the heaters' specs are pretty loose. 

An old ATX pc makes a good high current supply if you want to go down that rabbit hole. That was my first attempt at getting faster warmups before I realized the firmware prevented it.

One more thing: double check the connector itself; mine broke internally once and would start up since the pieces were touching, but would shut down when I tried to use it or wiggle the connector. A resoldered  it and it was fixed.
